### ISSUE TITLE
Unbounded Label Values

### ISSUE THREAD

**Issue by vinnicius** [labbsr0x/bb-protocol#5]
As presented in Prometheus [Metrics and Label Naming](https://prometheus.io/docs/practices/naming/#labels):

> CAUTION: Remember that every unique combination of key-value label pairs represents a new time series, which can dramatically increase the amount of data stored. Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values.

This is critical for Prometheus performance, with impact in both storage and memory consumption.

Speaking specifically of the label `errorMessages`, wouldn't it be healthier to classify errors into something more _characteristics-prone_ (as database|network|business)?

**Comment by vinnicius** [comment]
> I believe that great power comes with great responsibility, and we should just import this alert from Prometheus's guidelines to Big Brother's guidelines, without enforcing any policy on label's values bounds.

You're right. As this was a developer request, this should only be a warning.

More than an obligation, it may be important to mention in what ways a developer can take advantage of those guidelines. For example, using the same messages across all your applications makes it easy to correlate data of those metrics.

**Comment by eabili0** [comment]
Great!

@vinnicius, I welcome you to make such contribution yourself!! If you'd like, please open a Pull Request with changes to our `README.md` file, adding the aforementioned alert and error message best practices!
